README.md

```diff
@@ -8,10 +8,24 @@ parts of the [Whitehall](https://github.com/alphagov/whitehall) application.
 
 ![A  Case Study](https://raw.githubusercontent.com/alphagov/government-frontend/master/docs/assets/case-study-screenshot.png)
 
-## Live examples
-
-- [gov.uk/government/case-studies/2013-elections-in-swaziland](https://www.gov.uk/government/case-studies/2013-elections-in-swaziland)
-- [gov.uk/government/statistics/announcements/diagnostic-imaging-dataset-for-september-2015](https://www.gov.uk/government/statistics/announcements/diagnostic-imaging-dataset-for-september-2015)
+## Formats
+
+Not all formats that this app can handle are rendered by it in production.
+
+| Format | Example | Production status |
+|---|---|---|
+| Case study | [gov.uk/government/case-studies/2013-elections-in-swaziland](https://www.gov.uk/government/case-studies/2013-elections-in-swaziland) | Migrated |
+| Coming soon | | Rendered by Whitehall |
+| Detailed guide | | Rendered by Whitehall |
+| Document collection | | Rendered by Whitehall |
+| Fatality notice | | Rendered by Whitehall |
+| HTML Publication | | Rendered by Whitehall |
+| Publication | | Rendered by Whitehall |
+| Statistics announcement | [gov.uk/government/statistics/announcements/diagnostic-imaging-dataset-for-september-2015](https://www.gov.uk/government/statistics/announcements/diagnostic-imaging-dataset-for-september-2015) | Migrated |
+| Take part | [www.gov.uk/government/get-involved/take-part/become-a-councillor](https://www.gov.uk/government/get-involved/take-part/become-a-councillor) | Migrated |
+| Topical event about page | [gov.uk/government/topical-events/2014-overseas-territories-joint-ministerial-council/about](https://www.gov.uk/government/topical-events/2014-overseas-territories-joint-ministerial-council/about) | Migrated |
+| Unpublishing | | Rendered by Whitehall |
```

Comment:
It's a really murky area at present, there's work going on in Whitehall, Publishing API and Content Store at present to sort it out a bit. I don't _think_ it'll be a format by itself in the end, and I also don't think it'll end up being rendered by `government-frontend`. It can stay here for the moment though while it gets sorted out.